app/src/main/res/layout/activity_meal.xml

```diff
@@ -1,48 +1,58 @@
-        android:text="Order Meal"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
-
-    <ImageView
-        android:id="@+id/mealImage"
-        android:layout_width="229dp"
-        android:layout_height="147dp"
-        android:layout_marginTop="96dp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <TextView
-        android:id="@+id/mealName"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="12dp"
-        android:text="TextView"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/mealImage" />
-
-    <TextView
-        android:id="@+id/mealDesc"
-        android:layout_width="286dp"
-        android:layout_height="226dp"
-        android:layout_marginBottom="80dp"
-        android:text="TextView"
-        app:layout_constraintBottom_toTopOf="@+id/button"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.496"
-        app:layout_constraintStart_toStartOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:gravity="center">
+
+            <TextView
+                android:id="@+id/mealName"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="100dp"
+                android:layout_marginBottom="12dp"
+                android:text="TextView"/>
+
+            <ImageView
+                android:id="@+id/mealImage"
+                android:layout_width="229dp"
+                android:layout_height="147dp"
+                android:layout_marginBottom="12dp"
+                android:layout_marginTop="12dp"/>
+
+            <TextView
+                android:id="@+id/mealDesc"
+                android:layout_width="286dp"
+                android:layout_height="100dp"
+                android:layout_marginTop="12dp"
+                android:layout_marginBottom="30dp"
+                android:text="TextView"/>
+
+            <ListView
+                android:id="@+id/allergensMeal"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"/>
```

Comment:
Maybe you should change this in the next sprint into an HorizontalScrollView so it can look better.